### ISSUE TITLE
ffi 1.14.x  error installing on Ubuntu                         /usr/bin/ld:libffi.map:15: syntax error in VERSION script

### ISSUE THREAD

**Issue by joelxxx** [ffi/ffi#863]
Version 1.13.0 installed with no errors.
1.14.0 and 1.14.1 both threw the error shown below.

```
Fetching ffi 1.14.1
Installing ffi 1.14.1 with native extensions
Gem::Ext::BuildError: ERROR: Failed to build gem native extension.

    current directory: /home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c

/usr/bin/ruby2.7 -I /usr/lib/ruby/2.7.0 -r ./siteconf20201221-74406-1e2c1t6.rb extconf.rb
checking for ffi.h... no
checking for ffi.h in
/usr/local/include,/usr/include/ffi,/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/ffi,/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/ffi...
no
checking for whether -Wl,--exclude-libs,ALL is accepted as LDFLAGS... yes
creating extconf.h
creating Makefile

current directory: /home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c
make "DESTDIR=" clean

current directory: /home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c
make "DESTDIR="
Configuring libffi
make -C "/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi-x86_64-linux-gnu"
make[1]: Entering directory '/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi-x86_64-linux-gnu'
make  all-recursive
make[2]: Entering directory '/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi-x86_64-linux-gnu'
Making all in include
make[3]: Entering directory '/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi-x86_64-linux-gnu/include'
make[3]: Nothing to be done for 'all'.
make[3]: Leaving directory '/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi-x86_64-linux-gnu/include'
Making all in testsuite
make[3]: Entering directory '/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi-x86_64-linux-gnu/testsuite'
make[3]: Nothing to be done for 'all'.
make[3]: Leaving directory '/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi-x86_64-linux-gnu/testsuite'
Making all in man
make[3]: Entering directory '/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi-x86_64-linux-gnu/man'
make[3]: Nothing to be done for 'all'.
make[3]: Leaving directory '/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi-x86_64-linux-gnu/man'
make[3]: Entering directory '/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi-x86_64-linux-gnu'
/bin/bash ./libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi  -I.
-I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/include -Iinclude -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src   -Wall -fexceptions
-c -o src/prep_cif.lo /home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src/prep_cif.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi -I.
-I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/include -Iinclude -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src -Wall -fexceptions -c
/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src/prep_cif.c  -fPIC -DPIC -o src/.libs/prep_cif.o
/bin/bash ./libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi  -I.
-I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/include -Iinclude -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src   -Wall -fexceptions
-c -o src/types.lo /home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src/types.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi -I.
-I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/include -Iinclude -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src -Wall -fexceptions -c
/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src/types.c  -fPIC -DPIC -o src/.libs/types.o
/bin/bash ./libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi  -I.
-I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/include -Iinclude -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src   -Wall -fexceptions
-c -o src/raw_api.lo /home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src/raw_api.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi -I.
-I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/include -Iinclude -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src -Wall -fexceptions -c
/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src/raw_api.c  -fPIC -DPIC -o src/.libs/raw_api.o
/bin/bash ./libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi  -I.
-I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/include -Iinclude -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src   -Wall -fexceptions
-c -o src/java_raw_api.lo /home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src/java_raw_api.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi -I.
-I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/include -Iinclude -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src -Wall -fexceptions -c
/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src/java_raw_api.c  -fPIC -DPIC -o src/.libs/java_raw_api.o
/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src/java_raw_api.c: In function ‘ffi_java_raw_call’:
/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src/java_raw_api.c:317:3: warning: ‘ffi_java_raw_to_ptrarray’ is deprecated [-Wdeprecated-declarations]
  317 |   ffi_java_raw_to_ptrarray (cif, raw, avalue);
      |   ^~~~~~~~~~~~~~~~~~~~~~~~
/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src/java_raw_api.c:76:1: note: declared here
   76 | ffi_java_raw_to_ptrarray (ffi_cif *cif, ffi_java_raw *raw, void **args)
      | ^~~~~~~~~~~~~~~~~~~~~~~~
/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src/java_raw_api.c: In function ‘ffi_java_translate_args’:
/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src/java_raw_api.c:328:3: warning: ‘ffi_java_raw_size’ is deprecated [-Wdeprecated-declarations]
  328 |   ffi_java_raw *raw = (ffi_java_raw*)alloca (ffi_java_raw_size (cif));
      |   ^~~~~~~~~~~~
/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src/java_raw_api.c:45:1: note: declared here
   45 | ffi_java_raw_size (ffi_cif *cif)
      | ^~~~~~~~~~~~~~~~~
/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src/java_raw_api.c:331:3: warning: ‘ffi_java_ptrarray_to_raw’ is deprecated [-Wdeprecated-declarations]
  331 |   ffi_java_ptrarray_to_raw (cif, avalue, raw);
      |   ^~~~~~~~~~~~~~~~~~~~~~~~
/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src/java_raw_api.c:157:1: note: declared here
  157 | ffi_java_ptrarray_to_raw (ffi_cif *cif, void **args, ffi_java_raw *raw)
      | ^~~~~~~~~~~~~~~~~~~~~~~~
/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src/java_raw_api.c: In function ‘ffi_prep_java_raw_closure’:
/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src/java_raw_api.c:369:3: warning: ‘ffi_prep_java_raw_closure_loc’ is deprecated [-Wdeprecated-declarations]
  369 |   return ffi_prep_java_raw_closure_loc (cl, cif, fun, user_data, cl);
      |   ^~~~~~
/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src/java_raw_api.c:337:1: note: declared here
  337 | ffi_prep_java_raw_closure_loc (ffi_java_raw_closure* cl,
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/bin/bash ./libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi  -I.
-I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/include -Iinclude -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src   -Wall -fexceptions
-c -o src/closures.lo /home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src/closures.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi -I.
-I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/include -Iinclude -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src -Wall -fexceptions -c
/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src/closures.c  -fPIC -DPIC -o src/.libs/closures.o
/bin/bash ./libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi  -I.
-I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/include -Iinclude -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src   -Wall -fexceptions
-c -o src/x86/ffi64.lo /home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src/x86/ffi64.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi -I.
-I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/include -Iinclude -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src -Wall -fexceptions -c
/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src/x86/ffi64.c  -fPIC -DPIC -o src/x86/.libs/ffi64.o
/bin/bash ./libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi  -I.
-I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/include -Iinclude -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src  -I.
-I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/include -Iinclude -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src  -c -o
src/x86/unix64.lo /home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src/x86/unix64.S
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi -I.
-I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/include -Iinclude -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src -I.
-I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/include -Iinclude -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src -c
/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src/x86/unix64.S  -fPIC -DPIC -o src/x86/.libs/unix64.o
/bin/bash ./libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi  -I.
-I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/include -Iinclude -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src   -Wall -fexceptions
-c -o src/x86/ffiw64.lo /home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src/x86/ffiw64.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi -I.
-I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/include -Iinclude -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src -Wall -fexceptions -c
/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src/x86/ffiw64.c  -fPIC -DPIC -o src/x86/.libs/ffiw64.o
/bin/bash ./libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi  -I.
-I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/include -Iinclude -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src  -I.
-I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/include -Iinclude -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src  -c -o
src/x86/win64.lo /home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src/x86/win64.S
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi -I.
-I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/include -Iinclude -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src -I.
-I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/include -Iinclude -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src -c
/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src/x86/win64.S  -fPIC -DPIC -o src/x86/.libs/win64.o
gcc -DHAVE_CONFIG_H -I. -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi  -I. -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/include
-Iinclude -I/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/src   -Wall -fexceptions -DX86_64 -E -x assembler-with-cpp -o libffi.map
/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/libffi.map.in
/bin/bash ./libtool  --tag=CC   --mode=link gcc  -Wall -fexceptions   -o libffi_convenience.la  src/prep_cif.lo src/types.lo src/raw_api.lo src/java_raw_api.lo src/closures.lo   src/x86/ffi64.lo
src/x86/unix64.lo src/x86/ffiw64.lo src/x86/win64.lo
libtool: link: rm -fr  .libs/libffi_convenience.a(D)
libtool: link: ar cru .libs/libffi_convenience.a src/.libs/prep_cif.o src/.libs/types.o src/.libs/raw_api.o src/.libs/java_raw_api.o src/.libs/closures.o src/x86/.libs/ffi64.o src/x86/.libs/unix64.o
src/x86/.libs/ffiw64.o src/x86/.libs/win64.o
ar: `u' modifier ignored since `D' is the default (see `U')
libtool: link: ranlib .libs/libffi_convenience.a
libtool: link: ( cd ".libs" && rm -f "libffi_convenience.la" && ln -s "../libffi_convenience.la" "libffi_convenience.la" )
/bin/bash ./libtool  --tag=CC   --mode=link gcc  -Wall -fexceptions -no-undefined -version-info `grep -v '^#'
/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi/libtool-version` -Wl,--version-script,libffi.map    -o libffi.la -rpath /usr/local/lib/../lib src/prep_cif.lo src/types.lo
src/raw_api.lo src/java_raw_api.lo src/closures.lo  src/x86/ffi64.lo src/x86/unix64.lo src/x86/ffiw64.lo src/x86/win64.lo
libtool: link: gcc -shared  -fPIC -DPIC  src/.libs/prep_cif.o src/.libs/types.o src/.libs/raw_api.o src/.libs/java_raw_api.o src/.libs/closures.o src/x86/.libs/ffi64.o src/x86/.libs/unix64.o
src/x86/.libs/ffiw64.o src/x86/.libs/win64.o    -Wl,--version-script -Wl,libffi.map   -Wl,-soname -Wl,libffi.so.8 -o .libs/libffi.so.8.1.0
/usr/bin/ld:libffi.map:15: ignoring invalid character `"' in script
/usr/bin/ld:libffi.map:15: syntax error in VERSION script
collect2: error: ld returned 1 exit status
make[3]: *** [Makefile:1069: libffi.la] Error 1
make[3]: Leaving directory '/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi-x86_64-linux-gnu'
make[2]: *** [Makefile:1348: all-recursive] Error 1
make[2]: Leaving directory '/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi-x86_64-linux-gnu'
make[1]: *** [Makefile:623: all] Error 2
make[1]: Leaving directory '/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi-x86_64-linux-gnu'
make: *** [libffi.mk:9: "/home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1/ext/ffi_c/libffi-x86_64-linux-gnu"/.libs/libffi_convenience.a] Error 2

make failed, exit code 2

Gem files will remain installed in /home/deploy/myapp/vendor/bundle/ruby/2.7.0/gems/ffi-1.14.1 for inspection.
Results logged to /home/deploy/myapp/vendor/bundle/ruby/2.7.0/extensions/x86_64-linux/2.7.0/ffi-1.14.1/gem_make.out

An error occurred while installing ffi (1.14.1), and Bundler cannot continue.
Make sure that `gem install ffi -v '1.14.1' --source 'https://rubygems.org/'` succeeds before bundling.

In Gemfile:
  bootstrap was resolved to 4.5.3, which depends on
    sassc-rails was resolved to 2.1.2, which depends on
      sassc was resolved to 2.4.0, which depends on
        ffi
```

**Comment by RaphaelPour** [comment]
I can confirm that `1.14.1` is broken with Ubuntu 20.04.

**Comment by larskanis** [comment]
I can reproduce this issue and will fix and release ffi-1.14.2 shortly.

**Comment by larskanis** [comment]
ffi-1.14.2 is out and fixes this issue.
